src/App.css

```
@@ -27,3 +27,19 @@ tr:hover {
 .departureTime {
   text-align: left;
 }
+
+.ubahn-badge, .sbahn-badge, .train-badge {
+    padding: 3px 7px;
+    border-radius: 6px;
```

If we avoid vertical padding then more of the screen area has useful information on it. 
```suggestion
    padding: 0px 4px;
    font-weight: bold;
    color: white;
```